MessageBird/Objects/Request.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+using System.Text;
+
+namespace MessageBird.Objects
+{
+    public class Request
+    {
+        public string Timestamp { get; }
+        public string QueryParameters { get; }
+        public byte[] Data { get; }
+
+        private const char QueryParametersDelimiter = '&';
+
+        /**
+         * Constructs a new request instance.
+         *
+         * @param timestamp Timestamp provided in the MessageBird-Request-Timestamp
+         *                  header.
+         * @param queryParameters Query parameters in abc=foo&def=ghi format.
+         * @param data Raw body of this request.
+         */
+        public Request(string timestamp, string queryParameters, byte[] data) {
+            if (string.IsNullOrEmpty(timestamp)) {
+                throw new ArgumentNullException(nameof(timestamp));
```

Comment:
As per the README, we need to support .NET Framework >=4.0 and .NET Standard 2.0 - I think we can't use `nameof`?

Reply:
Good point! Thanks!